libc-test/build.rs

```diff
@@ -333,6 +356,12 @@ fn main() {
             "ptrace" |
             "sigaltstack" if rumprun => true,
 
+            // uses size_t instead of socklen_t
+            "getnameinfo" if dragonfly => true,
```

Comment:
I have to check this. DragonFly uses socklen_t, which probably is the same as size_t.